### PR TITLE
Implement nightly CI

### PR DESCRIPTION
**Pull request by prabalsharma**

### CODE REVIEW

File: qa-pipelines/README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Table of Contents
   * [Additional considerations](#additional-considerations)
     * [Deploy a pipeline which does a non-upgrade test of a custom bundle (which is neither an RC or a release)](#deploy-a-pipeline-which-does-a-non-upgrade-test-of-a-custom-bundle-not-an-rc-or-a-release)
     * [Continue a test suite from where a previous build left off](#continue-a-test-suite-from-where-a-previous-build-left-off)
+  * [Dev Nightly Upgrades CI](
+  #dev-nightly-upgrades-ci)
```
Comment (HartS) [inline qa-pipelines/README.md]: unnecessary line break

File: qa-pipelines/README.md [hunk 2]
```diff
@@ -109,3 +110,21 @@ The composability of the pipeline tasks means there are some interesting things
...
+# Dev Nightly Upgrades CI
+
+Nightly Builds are builds which happen every night from develop branch of scf. The build lands in s3://cap-release-archives/nightly/
+The idea here is to test bare minimum of these nightly builds, i.e.
+1. We want to make sure `helm install` and `helm upgrades` are not broken due to any changes
+2. Also, catch well in advance, if any changes to scf have broken qa-pipelines
+
+Dev ECP pool: https://github.com/SUSE/cf-ci-pools/tree/cap-ci-kube-hosts
+
+Concourse config for Dev ECP pool: [config-capci-nightly.yml](config-capci-nightly.yml)
```
Comment (HartS) [inline qa-pipelines/README.md]: Not accurate anymore, just mention `--nightly` flag

File: qa-pipelines/README.md [hunk 3]
```diff
@@ -109,3 +110,21 @@ The composability of the pipeline tasks means there are some interesting things
...
+
+Example command to deploy CI on concourse:
+
+`./set-pipeline -t provo -p Official-DEV-Nightly-Upgrades --pool=capci --nightly pipeline-presets/cap-qa-upgrades-lite.yml`
+
+[cap-qa-upgrades-lite.yml](pipeline-presets/cap-qa-upgrades-lite.yml) is more than enough to accomplish our goals here
```
Comment (HartS) [inline qa-pipelines/README.md]: I'd recommend changing this to [pipeline-presets/cap-qa-upgrades-lite.yml](pipeline-presets/cap-qa-upgrades-lite.yml) for clarity

File: qa-pipelines/README.md [hunk 4]
```diff
@@ -109,3 +110,21 @@ The composability of the pipeline tasks means there are some interesting things
...
+
+
+# Dev Nightly Upgrades CI
+
+Nightly Builds are builds which happen every night from develop branch of scf. The build lands in s3://cap-release-archives/nightly/
+The idea here is to test bare minimum of these nightly builds, i.e.
+1. We want to make sure `helm install` and `helm upgrades` are not broken due to any changes
+2. Also, catch well in advance, if any changes to scf have broken qa-pipelines
+
+Dev ECP pool: https://github.com/SUSE/cf-ci-pools/tree/cap-ci-kube-hosts
```
Comment (HartS) [inline qa-pipelines/README.md]:
```
The `cap-ci` pool [here](https://github.com/SUSE/cf-ci-pools/tree/cap-ci-kube-hosts), composed of kube clusters on ECP, will be our dedicated pool for nightly testing.
```

File: qa-pipelines/README.md [hunk 5]
```diff
@@ -109,3 +110,21 @@ The composability of the pipeline tasks means there are some interesting things
 In order to do this, replace the CAP bundle URLs in the pipeline config, enable *only* the pre-upgrade tests, and deploy the pipeline. The deploy will use the bundle from the `cap-sle-url` or `cap-opensuse-url` settings in the pipeline config.
 ## Continue a test suite from where a previous build left off.
 Sometimes running tests may fail for timing-related reasons which may be intermittent. If this happens, and you want to try to re-run the test and continue the build from where it left if, you can deploy a new pipeline with only the failed test and following tasks enabled, unlock the config which was used, and run a build from the new pipeline
+
+
+# Dev Nightly Upgrades CI
+
+Nightly Builds are builds which happen every night from develop branch of scf. The build lands in s3://cap-release-archives/nightly/
```
Comment (HartS) [inline qa-pipelines/README.md]:
```
... and will be picked up by any unpaused pipeline deployed with the `--nightly` flag with an available pool resource
```

Comment (prabalsharma) [reply qa-pipelines/README.md]: Dev nightly upgrades CI has nothing to do with any pool, other than capci

File: qa-pipelines/README.md (hunk 4 above)
Comment (prabalsharma) [reply qa-pipelines/README.md]: Its not QA's dedicated pool. Its on CAPCI project on ecp which is only for development work

Comment (HartS) [reply qa-pipelines/README.md]: If it's just one of several pools which may be used for nightly pipelines, I think we should remove this line altogether then.

**Comment by HartS** [comment]
LGTM now

**Pull request merged**